code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -1,6 +1,10 @@
 //Refer to life.dm for caller
 
 /mob/living/carbon/human/proc/handle_disabilities()
+	if(disabilities & ASTHMA)
+		if(prob(50))
```

Comment:
That's a bit much, don't you think ?

Reply:
It was purely for the sake of testing.  I left it with the intention of correcting it down to 0.5 after I figured out why the heck it was only working when tourettes was active.

Reply:
I think it should have a chance to occur every X minutes, rather than X% chance every Life()